versioned_docs/version-v2/framework-integration/angular.md

```diff
@@ -220,9 +220,8 @@ import { defineCustomElements } from 'stencil-library/loader';
   providers: [
     {
       provide: APP_INITIALIZER,
-      useFactory: () => {
-        return defineCustomElements();
-      },
+      useFactory: () => defineCustomElements,
```

Comment:
Double checking for my own understanding, why do we return `defineCustomElements` instead of calling it now? Does Angular looks through each of these providers and invoke the return value of `useFactory`?

Reply:
Calling it was unnecessary before. Angular will invoke the function itself. They show this usage on their docs: https://angular.io/guide/dependency-injection-providers#factory-providers-usefactory (also confirmed locally in the example app for the output targets repo that I'm setting up). 

Also had Tanner test in a stackblitz 🙂